jib-gradle-plugin/CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ All notable changes to this project will be documented in this file.
 ### Added
 
 - Image ID is now written to `build/jib-image.id` ([#1204](https://github.com/GoogleContainerTools/jib/issues/1204))
+- `jib.container.entrypoint = ['INHERIT']` allows inheriting `ENTRYPOINT` and `CMD` from the base image. While inheriting `ENTRYPOINT`, you can also override `CMD` using `jib.container.args`.
```

Comment:
Just realized maybe this should take just a string `'INHERIT'` to indicate inherit rather than the string in a list?

Reply:
Now `container.entrypoint` can accept a `String`, in which case it will be auto-translated into a singleton list.